numpy/core/tests/test_scalarmath.py

```diff
@@ -408,15 +408,16 @@ def test_longdouble_int(self):
             assert_raises(OverflowError, int, x)
             assert_equal(len(sup.log), 1)
 
-    @dec.knownfailureif(not IS_PYPY)
+    @dec.knownfailureif(not IS_PYPY,
+        "__int__ is not the same as int in cpython (gh-9972)")
```

Comment:
Not quite PEP8, but looks better. Lots of these around, so not going to worry about it.